tests/component/_task_modules/test_triggers.py

```diff
@@ -14,11 +16,22 @@ def ai_voltage_field_daq_task(task, sim_time_aware_9215_device):
     yield task
 
 
+@pytest.fixture()
+def ci_count_edges_task(task, sim_9185_device) -> nidaqmx.Task:
+    device_name = sim_9185_device.name
+    chan = task.ci_channels.add_ci_count_edges_chan(device_name + "/_ctr0")
```

Comment:
I find fstrings to be cleaner

```suggestion
    chan = task.ci_channels.add_ci_count_edges_chan(f"{sim_9185_device.name}/_ctr0")
```